### DISCUSSION TITLE
Use rover in azure dev ops / self hosted agents

### DISCUSSION THREAD

**Discussion by florian-ried** [aztfmod/terraform-azurerm-caf#1147] [Q&A]
Hello, I played through the "Single Subscription" demo perfectly: https://aztfmod.github.io/documentation/docs/azure-landing-zones/landingzones/platform/single%20reuse/elsz-single-reuse
All rover commands were executed locally on my machine. I'm now wondering how to build this in an Azure DevOps environment. I am also looking for how the self-hosted agents can be set up. Are there any helpful instructions, videos, etc. for beginners? Many thanks in advance.

**Comment by Raida95** [comment]
Hello,

I'm actually in the same situation and trying to find relevant documentation or guidance. Were you able to find any helpful resources to solve your problem? Your insights would be greatly appreciated. Many thanks in advance!
